airflow/executors/kubernetes_executor.py

```diff
@@ -397,7 +416,9 @@ def process_status(self, pod_id, status, labels, resource_version):
 
 
 class AirflowKubernetesScheduler(LoggingMixin):
+    """Airflow Scheduler for Kubernetes"""
     def __init__(self, kube_config, task_queue, result_queue, kube_client, worker_uuid):
+        """Creates the executor"""
```

Comment:
```suggestion
```
It is not necessary. We should keep the documentation in the class description

Reply:
Done